_gsocproposals/2020/proposal_ATLAS-Visualization.md

```diff
@@ -0,0 +1,60 @@
+---
+title: GeoModelExplorer, interactive 3D geometry visualization - Development of a visualization tool to interactively explore the  geometry of a HEP detector
+layout: gsoc_proposal
+project: ATLAS
```

Comment:
Is there an ATLAS project described somewhere yet? If not, you have to add one there.

Reply:
Thanks a lot for your commet, @aflp91! 
I didn't notice that this was the first proposal as "ATLAS". 

I will add that now.

Reply:
Done.
Please, let me know if further additions or changes are needed.
Thanks, -- Riccardo.